ResearchKit/Common/ORKSkin.m

```diff
@@ -113,21 +113,48 @@ ORKScreenType ORKGetScreenTypeForBounds(CGRect bounds) {
     return screenType;
 }
 
-ORKScreenType ORKGetScreenTypeForWindow(UIWindow *window) {
+ORKScreenType ORKGetHorizontalScreenTypeForBounds(CGRect bounds) {
+    ORKScreenType screenType = ORKScreenTypeiPhone6;
+    CGFloat minimumDimension = MIN(bounds.size.width, bounds.size.height);
+    if (minimumDimension < ORKiPhone4ScreenSize.width + 1) {
+        screenType = ORKScreenTypeiPhone4;
+    } else if (minimumDimension < ORKiPhone5ScreenSize.width + 1) {
+        screenType = ORKScreenTypeiPhone5;
+    } else if (minimumDimension < ORKiPhone6ScreenSize.width + 1) {
+        screenType = ORKScreenTypeiPhone6;
+    } else if (minimumDimension < ORKiPhone6PlusScreenSize.width + 1) {
+        screenType = ORKScreenTypeiPhone6Plus;
+    } else {
+        screenType = ORKScreenTypeiPad;
+    }
+    return screenType;
+}
+
+ORKScreenType ORKGetVerticalScreenTypeForWindow(UIWindow *window) {
     if (!window) {
         window = [[[UIApplication sharedApplication] windows] firstObject];
```

Comment:
Sorry for the late comment: wouldn't using `[UIApplication sharedApplication].keyWindow` here be more correct?